### PR TITLE
B-19722

### PR DESCRIPTION
**Pull request by paulstonebraker**
## Summary

USCG customers must enter EMPLID on account creation. This value is editable when a move is in draft state.

### How to test

1. Create a new customer
2. Select Coast Guard as branch and enter EMPLID
3. Go back and forth between pages and try to break (change branches, enter letters, etc)
4. If not broken advance through profile and orders creation until you have a move in draft. Access your profile screen and edit your EMPLID and try to break
5. If not broken, create a new customer and confirm that EMPLID must be unique

### CODE REVIEW

File: src/components/Customer/ServiceInfoForm/ServiceInfoForm.test.jsx [hunk 1]
```diff
@@ -234,5 +235,70 @@ describe('ServiceInfoForm', () => {
...
+    it('shows an error message if trying to submit an invalid form', async () => {
+      isBooleanFlagEnabled.mockImplementation(() => Promise.resolve(true));
+      testPropsWithEdipi.initialValues.affiliation = 'COAST_GUARD';
+      render(<ServiceInfoForm {...testPropsWithEdipi} />);
+
+      // Touch required fields to show validation errors
+      // Skip branch because Coast Guard needs to be selected for EMPLID to appear
+      await userEvent.click(screen.getByLabelText('First name'));
+      await userEvent.click(screen.getByLabelText('Last name'));
+      await userEvent.click(screen.getByLabelText('DoD ID number'));
```
Comment (pambecker) [inline src/components/Customer/ServiceInfoForm/ServiceInfoForm.test.jsx:291]: This DoDID line is new, not in your INT PR.

Comment (paulstonebraker) [reply src/components/Customer/ServiceInfoForm/ServiceInfoForm.test.jsx]: Thanks, good eyes 🔍 🕵️‍♀️

**Review by pambecker — APPROVED** [review]

**Comment by pambecker** [comment]
Just remembered, be sure to update the DB tracking spreadsheet for this story.

**Review by loganwc — APPROVED** [review]
only difference is the okta dodid feature flag has a couple added rows but I don't see any real changes. Good to go!

**Pull request merged**